src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardDetails.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
-import { Text, Flex, Box } from '@pancakeswap/uikit'
+import { Text, Flex, Box, Skeleton } from '@pancakeswap/uikit'
 import { PublicIfoData } from 'hooks/ifo/types'
 import { useTranslation } from 'contexts/Localization'
 import { Ifo, PoolIds } from 'config/constants/types'
 import { getBalanceNumber, formatNumber } from 'utils/formatBalance'
 import { SkeletonCardDetails } from './Skeletons'
+import ApyButton from '../../../../Farms/components/FarmCard/ApyButton'
```

Comment:
Looks like this is not used anymore.

Reply:
It is a leftover from wrong copy paste :) , I removed it.